static/app/components/sidebar/index.tsx

```diff
@@ -220,17 +223,82 @@ function Sidebar() {
     />
   );
 
-  const issues = hasOrganization && (
+  let issues = hasOrganization && (
     <SidebarItem
       {...sidebarItemProps}
       icon={<IconIssues />}
       label={<GuideAnchor target="issues">{t('Issues')}</GuideAnchor>}
       to={`/organizations/${organization.slug}/issues/`}
       search="?referrer=sidebar"
       id="issues"
-      hasNewNav={hasNewNav}
     />
   );
+  if (hasNewNav && hasOrganization) {
+    issues = (
+      <SubnavMenu
```

Comment:
Most of the functional markup changes are handled by a new `SubnavMenu` component, which projects children into the secondary sidebar panel as needed